PocketKit/Sources/PocketKit/Article/ReadableViewModel/SavedItemViewModel.swift

```diff
@@ -145,12 +149,12 @@ extension SavedItemViewModel {
         ]
     }
 
-    private func favorite() {
+    internal func favorite() {
         source.favorite(item: item)
         track(identifier: .itemFavorite)
     }
 
-    private func unfavorite() {
+    internal func unfavorite() {
```

Comment:
```suggestion
    func unfavorite() {
```

```suggestion
    func favorite() {
```

Internal by default; no need to explicitly mark it as such.